appmap/pytest.py

```diff
@@ -113,13 +113,13 @@ def pytest_runtestloop(session):
 
 
 @pytest.hookimpl(hookwrapper=True)
-def pytest_pyfunc_call(pyfuncitem):
+def pytest_runtest_call(item):
```

Comment:
Was there some reason this needed to be changed? Hooking`pytest_pyfunc_call` seemed like it would best guarantee that we avoided instrumenting any setup or teardown code.

Reply:
The problem is `pytest_pyfunc_call` doesn't get called for unittest cases. (BTW, unfortunately `pytest_runtest_call` does end up encompassing setup and teardown for unittest cases. I suppose for now it's good enough. We'll improve it in #5.)

Reply:
Is there something else in this PR that depends on this change? Or is this just a convenience? If the latter, is there a way we could make it opt-in instead?

Reply:
Without this change HTTP request capture would be pretty much dead code, since *django.test.TestCase* (and I assume what most people use to test HTTP requests) is a *unittest* subclass.